content/blog/2018-10-01-create-react-app-v2.md

```diff
@@ -28,7 +28,8 @@ Here's a short summary of what's new in this release:
 * 🐈 You can try the experimental [Yarn Plug'n'Play mode](https://github.com/yarnpkg/rfcs/pull/101) that removes `node_modules`.
 * 🕸 You can now [plug your own proxy implementation](https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#configuring-the-proxy-manually) in development to match your backend API.
 * 🚀 You can now use [packages written for latest Node versions](https://github.com/sindresorhus/ama/issues/446#issuecomment-281014491) without breaking the build.
-* 💄 You can now optionally get a smaller CSS bundle if you only plan to target modern browsers.
+* 💄 We added [PostCSS](https://github.com/csstools/postcss-preset-env#postcss-preset-env-) so you can use [new CSS features](https://cssdb.org/) in old browsers.
```

Comment:
I think ideally we should send a PR to https://preset-env.cssdb.org/features#all-property to allow linking to "stage 3" preset (currently it's just a select). Then we could link to that. Otherwise it's too ambiguous about which features we enable.

Reply:
I specified stage 3+ for now.